### PR TITLE
Add ask page

### PR DESCRIPTION
**Pull request by somatorio**

### CODE REVIEW

File: static/presentation/index.html [hunk 1]
```diff
@@ -23,18 +22,11 @@ <h2 class="subtitle">
...
-                        <h2 class="bd-link-name">
-                            <figure class="bd-link-figure">
-                                <span class="bd-link-counter"></span>
-                            </figure>
-                            {{ person }}
-                        </h2>
-                    </a>
-                </nav>
-            </div>
+            <ol>
```
Comment (sebawebber) [inline static/presentation/index.html:25]: nice refactor

**Review by sebastianwebber — APPROVED** [review]
ok for v1 ask option

File: static/js/app.js [hunk 2]
```diff
@@ -3,45 +3,94 @@ new Vue({
...
+        },
+        getHands: function (event) {
+            axios
+            .get('/api/hands')
+            .then(response => (this.handList = response.data.hands))
+            .catch(error => console.log(error))
+        },
+        saveWho: function () {
+            if (this.newWho !== localStorage.who) {
+                if (this.handList !== null && this.handList.indexOf(this.newWho) != -1) {
```
Comment (sebawebber) [inline static/js/app.js]: you can remove the `else` returning the values inside the `if` block.

File: static/js/app.js [hunk 3]
```diff
@@ -3,45 +3,94 @@ new Vue({
     data() {
         return {
             handList: null,
-            counter: 0
+            who: '',
```
Comment (sebawebber) [inline static/js/app.js]: rename var to an useful name like who to `inputName` or `inputWho`

File: static/js/app.js [hunk 4]
```diff
@@ -3,45 +3,94 @@ new Vue({
...
+            newWho: '',
```
Comment (sebawebber) [inline static/js/app.js]: rename var to an useful name like who to `localStorageName` or `localStorageWho`

File: static/js/app.js [hunk 5]
```diff
@@ -3,45 +3,94 @@ new Vue({
...
-                        data: bodyFormData,
-                        config: { headers: { 'Content-Type': 'application/x-www-form-urlencoded' } }
-                    })
-                        .then(function (response) {
-                            console.log(response);
-                        })
-                        .catch(function (error) {
-                            console.log(error);
-                        });
+            if (confirm == true) {
```
Comment (sebawebber) [inline static/js/app.js:11]: use same analogy here: return inside the `if`  block to avoid the usage of `else` block.

File: static/js/app.js [hunk 6]
```diff
@@ -3,45 +3,94 @@ new Vue({
...
+                this.$dialog.confirm({
+                    message: 'Remove from Queue?',
+                    onConfirm: () => this.remove(event, who, this)
+                });
+            } else {
+                this.remove(event, who, this);
+            }
+        },
+        remove: function (event, who, context) {
```
Comment (sebawebber) [inline static/js/app.js]: I suggest you merge both functions. they do the same except for the new URL.

File: static/js/app.js [hunk 7]
```diff
@@ -3,45 +3,94 @@ new Vue({
...
+                    console.log(error);
+                });
+        },
+        getHands: function (event) {
+            axios
+            .get('/api/hands')
+            .then(response => (this.handList = response.data.hands))
+            .catch(error => console.log(error))
+        },
+        saveWho: function () {
```
Comment (sebawebber) [inline static/js/app.js:43]: rename function to a useful name like `saveLocal` or `saveOnLocalStorage`

**Review by sebastianwebber — CHANGES_REQUESTED** [review]

File: static/js/app.js (hunk 3 above)
Comment (somatorio) [reply static/js/app.js]: It is EXACTLY what the name implies, who is using the client

File: static/js/app.js (hunk 7 above)
Comment (somatorio) [reply static/js/app.js:43]: it does EXACTLY what the name implies, save the name of whoever is using the client

**Pull request merged**